DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/BaseConversionToSpeckle.cs

```diff
@@ -8,30 +7,21 @@ namespace Speckle.Converters.RevitShared.ToSpeckle;
 // POC: maybe but could be generic abstract Converters.Common?
 public abstract class BaseConversionToSpeckle<THost, TSpeckle>
   : IHostObjectToSpeckleConversion,
-    IRawConversion<THost>,
     IRawConversion<THost, TSpeckle>
-  where TSpeckle : notnull
+  where TSpeckle : Base
 {
   public Base Convert(object target)
   {
-    return ConvertToBase((THost)target);
-  }
+    var result = RawConvert((THost)target);
```

Comment:
I don't suppose this hurts tbh

Reply:
I'm still deciding on this one. I just removed the unnecessary method nesting.